.github/workflows/ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  test-build:
+    name: 👷 Test & Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.17"
+
+      - uses: actions/checkout@v2
+
+      # - name: Test
```

Comment:
what's the plan with this?

Reply:
I don't have permission to add api keys to the repo, that why it's commented out. I'll remove it from this PR.

Reply:
let's do separate and ping me, I will add it 😉